tests/test_device.py

```diff
@@ -270,10 +627,9 @@ def test_device_gets_local_config(self, monkeypatch, tmpdir):
         dev = qml.device("honeywell.hqs", wires=2, machine=DUMMY_MACHINE)
 
         assert dev.shots == 99
-        assert API_HEADER_KEY in dev.header.keys()
-        assert dev.header[API_HEADER_KEY] == SOME_API_KEY
+        assert dev._user == SOME_API_KEY
 
-    def test_device_gets_api_key_default_config_directory(self, monkeypatch, tmpdir):
+    def test_device_gets_user_default_config_directory(self, monkeypatch, tmpdir):

```

Comment:
This test fails for me, locally, although I suspect it has something to do with the `# HACK: only works on linux` part. 😅

Reply:
Yes likely :smile: Here the name of the test and the email part was changed.